src/cryptoadvance/specter/rpc.py

```diff
@@ -264,8 +264,29 @@ def multi(self, calls: list, **kwargs):
         url = self.url
         if "wallet" in kwargs:
             url = url+"/wallet/{}".format(kwargs["wallet"])
-        r = requests.post(
-            url, data=json.dumps(payload), headers=headers, timeout=timeout)
+        r = None
+        if 'localhost:' not in url and '127.0.0.1:' not in url:
```

Comment:
why for all domains? it will certainly fail for any local network ip addresses.
Maybe better to do it only for onion?

```suggestion
        if '.onion' in url:
```

Reply:
ok